### PR TITLE
Added Composite Test Output

### PR DESCRIPTION
**Pull request by basvodde**
How about achieving combinations of output this way?

### CODE REVIEW

**Comment by arstrube** [comment]
Sadly, Travis CI conked out because of some makefile problem.

It looks like a clean solution.

Naturally I am partial towards my own, though (the original one, not the "simplified" one) :P

File: Makefile.am [hunk 1]
```diff
@@ -56,6 +57,7 @@ include_cpputestdir = $(includedir)/CppUTest
 include_cpputest_HEADERS = \
 	include/CppUTest/CommandLineArguments.h \
 	include/CppUTest/CommandLineTestRunner.h \
+	include/CppUTest/ComponsiteTestOutput.h \
```
Comment (arstrube) [inline Makefile.am]: Componsite, huh ;-)

**Comment by arstrube** [comment]
Looks good; personally, I wouldn't extract it into separate files and be done without attracting all this extra work, bringing automake, cmake, and VS up to scratch (not to talk about the "other" platforms...

**Comment by arstrube** [comment]
Oh, but it doesn't "work" :-(

At least, not yet......

**Comment by basvodde** [comment]
Hrm, messing up git rebase. Closing and trying again.

**Pull request closed without merging**